packages/@expo/prebuild-config/src/plugins/unversioned/expo-splash-screen/getIosSplashConfig.ts

```diff
@@ -32,6 +33,7 @@ export function getIosSplashConfig(config: ExpoConfig): IOSSplashConfig | null {
       image,
       resizeMode: splash.resizeMode ?? defaultResizeMode,
       backgroundColor: splash.backgroundColor ?? defaultBackgroundColor,
+      fadeDurationMs: splash['fadeDurationMs'] ?? 0,
```

Comment:
the ios plugin here still references the legacy splash config. didn't see the code to handle custom splash props in this pr.

Reply:
Good catch -- in fact custom splash fadeDurationMs works, but setting it in the "splash" object in app.json does not work.

Reply:
This code was already in `getIosSplashScreen.ts` so did not need to be modified for this PR.